test/PerpetualPositionRouter.t.sol

```diff
@@ -38,11 +38,7 @@ contract PositionRouterTest is Test, PerpetualContracts {
     );
     (bool okTwo,) = payable(vethPositionRouterAddr).call(
       abi.encodePacked(
-        uint8(5),
-        uint160(sqrtPriceLimitX96),
-        uint32(block.timestamp),
-        uint96(0),
-        uint96(oppositeAmountBound)
+        uint8(5), uint160(sqrtPriceLimitX96), uint32(block.timestamp), uint96(oppositeAmountBound)
```

Comment:
_(arbitrary comment location to start a thread)_

PR description says `Closes #29 #28` and it seems github will only recognize the first issue from this, to have it link both you need `Closes #29, closes #28`

Reply:
Ah okay thanks!